components/slack/sources/new-interactions/new-interactions.mjs

```diff
@@ -0,0 +1,83 @@
+import common from "../common/base.mjs";
+
+export default {
+  name: "New Interactions",
+  version: "0.0.1",
+  key: "new-interactions",
```

Comment:
should be something like `slack-new-interaction-event-received` (and the name of the file and folder should match)

Reply:
Renamed!